shopfloor_reception/services/reception.py

```diff
@@ -756,6 +918,12 @@ def done_action(self, picking_id, confirmation=False):
                 picking, message=self.msg_store.transfer_no_qty_done()
             )
         if not confirmation:
+            # Do not create a backorder if this is a shopfloor return.
+            if picking.is_shopfloor_return and self.work.menu.allow_return:
```

Comment:
We don't create backorder because it was created by shopfloor. We need this concept in other scenario where allow_move_create
```suggestion
            if picking.is_shopfloor_create and self.work.menu.allow_return:
```

Reply:
I'm about to open a PR for reception where I rename this field, just in case.